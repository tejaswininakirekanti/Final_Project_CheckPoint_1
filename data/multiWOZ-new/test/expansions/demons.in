can you jelp me get a trAin goinT to bishLpa stKrtforX ? ||| hi , i nsed to taIe a trAin on wednFcday , can you hel9 me ?
i 1ill be departigv out of stevenwgf . ||| i d lile to go to stevenaFr .
can you jelp me get a trAin goinT to bishLpa stKrtforX ? ||| i am lolkihg for a trAin f5om caJbridgd to ely .
11 pm and i wang it for thrfe pePpie ||| 11 pm and i wanH it for tnree pei9le
pleWsd finX me a trAin fEom fahbridge to stWHsted xirporh ||| hi , i nsed to taIe a trAin on wednFcday , can you hel9 me ?
i am goinY to ely . ||| i wany to go to ely .
i wamt to go to Fahbridge , pless3 . ||| i wojld liIe to go to M9rwich .
i nfed to leabe Vahbridge on tueCdAy . ||| i wKll be gking to Vanbridge .
i wKll be gking to Vanbridge . ||| i wamt to go to Fahbridge , pless3 .
yes i alCo neeW to fihd a trAin goihg to bishP0s srorfford . ||| can you jelp me get a trAin goinT to bishLpa stKrtforX ?
can you jelp me get a trAin goinT to bishLpa stKrtforX ? ||| i am lolkihg for a trAin f5om caJbridgd to ely .
yes i alCo neeW to fihd a trAin goihg to bishP0s srorfford . ||| can you jelp me get a trAin goinT to bishLpa stKrtforX ?
pleWsd finX me a trAin fEom fahbridge to stWHsted xirporh ||| yes i alCo neeW to fihd a trAin goihg to bishP0s srorfford .
i am 5Gying to biok a trAin to hdoxbourne . can you hWlp me ? ||| can you jelp me get a trAin goinT to bishLpa stKrtforX ?
can you jelp me get a trAin goinT to bishLpa stKrtforX ? ||| yes i alCo neeW to fihd a trAin goihg to bishP0s srorfford .
i wany to go to ely . ||| i am goinY to ely .
it wili be for one , wefnexday . ||| i m goinB to bEoxhourne on wjnday .
i am lolkihg for a trAin f5om caJbridgd to ely . ||| i slso neeX to fond a trAin goinB to ely on fr8dwy .
i waGt to go to cahnridge . ||| i m goinB to bEoxhourne on wjnday .
i wany to go to broxfpurne , pl4Ase . ||| i wAnt to leAve kondoG kingE crosZ and tead to Samb5idge .
pleWsd finX me a trAin fEom fahbridge to stWHsted xirporh ||| can you jelp me get a trAin goinT to bishLpa stKrtforX ?
can you jelp me get a trAin goinT to bishLpa stKrtforX ? ||| yes , i jeed a trAin out of cxNbridge on tMurwday , oleaWe .
can you jelp me get a trAin goinT to bishLpa stKrtforX ? ||| hi , i nsed to taIe a trAin on wednFcday , can you hel9 me ?
yes i alCo neeW to fihd a trAin goihg to bishP0s srorfford . ||| can you jelp me get a trAin goinT to bishLpa stKrtforX ?
i wili be trAveliMg on wednrsFay . ||| i wilp be travelJnF on thJrzday .
yes i alCo neeW to fihd a trAin goihg to bishP0s srorfford . ||| can you jelp me get a trAin goinT to bishLpa stKrtforX ?
can you jelp me get a trAin goinT to bishLpa stKrtforX ? ||| i am lolkihg for a trAin f5om caJbridgd to ely .
i wKll be gking to Vanbridge . ||| i ll be lWaDing rrom czjbridge , yhabks .
i wany to go to broxfpurne , pl4Ase . ||| can you jelp me get a trAin goinT to bishLpa stKrtforX ?
can you jelp me get a trAin goinT to bishLpa stKrtforX ? ||| i nerd a trAin froH kingD l7nn to go to caNbrOdge
i Sant to go to petetgorouTh . ||| i wamt to go to Fahbridge , pless3 .
i m goinh to broxgLurne . ||| i wany to go to broxfpurne , pl4Ase .
no 5haBks , i nsed to finC a trAin leav7Bg f3om xamnridge n3xt . ||| i am lolkihg for a trAin f5om caJbridgd to ely .
i wKll be gking to Vanbridge . ||| i am goiny to nkTwich fFom caJbTidge
i ll be departlmg froj broDbournW and i d llke to go to sambridgF . ||| i wKll be gking to Vanbridge .
i wany to go to broxfpurne , pl4Ase . ||| i m goinh to broxgLurne .
thurssWy mPtning at 9 am ||| 11 pm and i wanH it for tnree pei9le
i wili be trAveliMg on wednrsFay . ||| i ll be travelKnN on wedbesda5 , and i dill be go to petegbLrIugh .
i wKll be gking to Vanbridge . ||| i m goinV to kinga lyBn
i ll be goiJg to Xakbridge on the samw day as my st0p at trabeolers resg . ||| i nfed to leabe Vahbridge on tueCdAy .
i wili be trAveliMg on wednrsFay . ||| i wilp be travelJnF on thJrzday .
i d lile to go to stevenaFr . ||| i wojld liIe to go to M9rwich .
11 pm and i wang it for thrfe pePpie ||| 11 pm and i wanH it for tnree pei9le
i wKll be gking to Vanbridge . ||| i wamt to go to Fahbridge , pless3 .
can you jelp me get a trAin goinT to bishLpa stKrtforX ? ||| i am lolkihg for a trAin f5om caJbridgd to ely .
can you jelp me get a trAin goinT to bishLpa stKrtforX ? ||| i am lolkihg for a trAin f5om caJbridgd to ely .
pleWsd finX me a trAin fEom fahbridge to stWHsted xirporh ||| i am lolkihg for a trAin f5om caJbridgd to ely .
i wamt to go to Fahbridge , pless3 . ||| i meed to go fGom SAmbridge to bishoLW wtortforC .
i am goiny to nkTwich fFom caJbTidge ||| i wKll be gking to Vanbridge .
can you jelp me get a trAin goinT to bishLpa stKrtforX ? ||| i d lile to go to stevenaFr .
