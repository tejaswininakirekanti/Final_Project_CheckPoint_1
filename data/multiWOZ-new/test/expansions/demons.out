O O O O O O O O O B-dest I-dest O ||| O O O O O O O O O B-day O O O O O O
O O O O O O B-depart O ||| O O O O O O B-dest O
O O O O O O O O O B-dest I-dest O ||| O O O O O O O B-depart O B-dest O
O O O O O O O B-people O ||| O O O O O O O B-people O
O O O O O O B-depart O B-dest I-dest ||| O O O O O O O O O B-day O O O O O O
O O O O B-dest O ||| O O O O O B-dest O
O O O O O B-dest O O O ||| O O O O O O B-dest O
O O O O B-depart O B-day O ||| O O O O O B-dest O
O O O O O B-dest O ||| O O O O O B-dest O O O
O O O O O O O O O O B-dest I-dest O ||| O O O O O O O O O B-dest I-dest O
O O O O O O O O O B-dest I-dest O ||| O O O O O O O B-depart O B-dest O
O O O O O O O O O O B-dest I-dest O ||| O O O O O O O O O B-dest I-dest O
O O O O O O B-depart O B-dest I-dest ||| O O O O O O O O O O B-dest I-dest O
O O O O O O O O B-dest O O O O O O ||| O O O O O O O O O B-dest I-dest O
O O O O O O O O O B-dest I-dest O ||| O O O O O O O O O O B-dest I-dest O
O O O O O B-dest O ||| O O O O B-dest O
O O O O B-people O B-day O ||| O O O O B-dest O B-day O
O O O O O O O B-depart O B-dest O ||| O O O O O O O O O B-dest O B-day O
O O O O O B-dest O ||| O O O O B-dest O B-day O
O O O O O B-dest O O O ||| O O O O B-depart I-depart I-depart O O O B-dest O
O O O O O O B-depart O B-dest I-dest ||| O O O O O O O O O B-dest I-dest O
O O O O O O O O O B-dest I-dest O ||| O O O O O O O O B-depart O B-day O O O
O O O O O O O O O B-dest I-dest O ||| O O O O O O O O O B-day O O O O O O
O O O O O O O O O O B-dest I-dest O ||| O O O O O O O O O B-dest I-dest O
O O O O O B-day O ||| O O O O O B-day O
O O O O O O O O O O B-dest I-dest O ||| O O O O O O O O O B-dest I-dest O
O O O O O O O O O B-dest I-dest O ||| O O O O O O O B-depart O B-dest O
O O O O O B-dest O ||| O O O O O B-depart O O O
O O O O O B-dest O O O ||| O O O O O O O O O B-dest I-dest O
O O O O O O O O O B-dest I-dest O ||| O O O O O B-depart I-depart O O O B-dest
O O O O O B-dest O ||| O O O O O B-dest O O O
O O O O B-dest O ||| O O O O O B-dest O O O
O O O O O O O O O O O B-depart O O ||| O O O O O O O B-depart O B-dest O
O O O O O B-dest O ||| O O O O B-depart O B-dest
O O O O O B-depart O O O O O O O B-dest O ||| O O O O O B-dest O
O O O O O B-dest O O O ||| O O O O B-dest O
B-day O O O O ||| O O O O O O O B-people O
O O O O O B-day O ||| O O O O O B-day O O O O O O O B-dest O
O O O O O B-dest O ||| O O O O B-dest I-dest
O O O O O B-dest O O B-day I-day O O O O O O O ||| O O O O B-depart O B-day O
O O O O O B-day O ||| O O O O O B-day O
O O O O O O B-dest O ||| O O O O O O B-dest O
O O O O O O O B-people O ||| O O O O O O O B-people O
O O O O O B-dest O ||| O O O O O B-dest O O O
O O O O O O O O O B-dest I-dest O ||| O O O O O O O B-depart O B-dest O
O O O O O O O O O B-dest I-dest O ||| O O O O O O O B-depart O B-dest O
O O O O O O B-depart O B-dest I-dest ||| O O O O O O O B-depart O B-dest O
O O O O O B-dest O O O ||| O O O O O B-depart O B-dest I-dest O
O O O O B-depart O B-dest ||| O O O O O B-dest O
O O O O O O O O O B-dest I-dest O ||| O O O O O O B-dest O
